src/components/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputProps {
+    onClick?: () => void;
+    className?: string;
+    title: string;
+  }
+
+const Input=({
+    className = '',
+    title,
+  }: InputProps)=> {
+  const [isChecked, setIsChecked] = useState(false);
+
+  return (
+    <>
+      <div className="flex gap-2">
+        <input
+          type="checkbox"
+          checked={isChecked}
+          onChange={() => setIsChecked((prev) => !prev)}
+          title={title}
+          className={'w-4 h-4 text-blue-600 ${className}'}
+        />
+        <label className="text-sm font-medium text-gray-900 dark:text-gray-300">{isChecked ? title:""}</label>
```

Comment:
If the checkbox is not checked, it might be better not to display the label (even if it is an empty string).

```suggestion
     {isChecked && <label className="text-sm font-medium text-gray-900 dark:text-gray-300">{title}</label>}
```

Reply:
Got it, made the changes !